### ISSUE TITLE
[BUG] Empty DNSSEC flag is sent during domain registration/transfer

### ISSUE THREAD

**Issue by soberoctober** [openprovider/OP-WHMCS7#88]
Reported in #58 

According to the settings of the Domain Controller, the module should be sending a false boolean value in the isDnssecEnabled field of the createDomainRequest and transferDomainRequest calls.

However, the setting is disregarded and the flag is being sent with an empty value:

```
Array
(
    [postValues] => <?xml version="1.0" encoding="UTF-8"?>
...
<nameServers><array><item><name>ns1.openprovider.nl</name><ip></ip></item><item><name>ns2.openprovider.be</name><ip></ip></item><item><name>ns3.openprovider.eu</name><ip></ip></item></array></nameServers><autorenew>off</autorenew><additionalData/><nsTemplateName></nsTemplateName><useDomicile>0</useDomicile><isPrivateWhoisEnabled>0</isPrivateWhoisEnabled><acceptPremiumFee>0</acceptPremiumFee><isDnssecEnabled></isDnssecEnabled><dnsmanagement></dnsmanagement></createDomainRequest></openXML>

)
```

*Use case*

1. A customer registers a domain name using the default Openprovider DNS 
2. Once the domain is registered, the customer changes the DNS to anything else (i.e. cloudflare).
3. Since DNSSEC is still active, the new DNS information doesn't become fully active and the domain doesn't respond to the new server.

*Expected result*

'0' value is enforced in the `<isDnssecEnabled></isDnssecEnabled>` field  of both createDomainRequest and transferDomainRequest calls.

**Comment by BenMeesters** [comment]
Whats taking so long two fix this. hire a programmer plz.

**Comment by soberoctober** [comment]
fixed in a538da4
